npc_combat_test.py

```diff
@@ -57,7 +57,7 @@ def test_clash_atk_blk(player_atk, player_blk, npc_atk, npc_blk, expected_player
     npc_hp_before_clash = npc.hp
     npc_mod = uuid4()
 
-    clash(player_atk, player_blk, player, npc_atk, npc_blk, npc_mod, npc)
+    clash(player_atk, player_blk, "foo", player, npc_atk, npc_blk, npc_mod, npc)
```

Comment:
Change `"foo"` to `uuid4()`.... imo this more clearly indicates that the value doesn't matter because a UUID4 is randomized whereas "foo" could technically be hardcoded somewhere and have meaning (*highly* unlikely a developer would use "foo" in real code, but doesn't hurt to be a stickler about these things).